src/object/srv_obj_migrate.c

```diff
@@ -1955,8 +2002,11 @@ punch_iod_pack(struct migrate_one *mrone, daos_iod_t *iod, daos_epoch_t eph)
 			return -DER_NOMEM;
 	}
 
+	if (obj_is_ec(obj))
+		boundary = obj_ec_tgt_nr(&obj->cob_oca) * iod->iod_size;
```

Comment:
Why contains the parity shards for boundary?

Reply:
Because the data/parity within different stripe has to use their own epoch, if we merge these extents cross the parity/stripe boundary, their epoch will have to merge as well, which might cause the data corruption during degrade fetch.  So we set boundary here to avoid the merging.

Sorry, really do not want to rerun the test unless there are obvious logic error in this patch. Thanks.  I will address these error in the next patch.

Reply:
My question is why the "boundary" is not "obj_ec_stripe_rec_nr(&obj->cob_oca);" that only contains data shards instead of why we need "boundary".

Reply:
ah, yes, you are right, it should be obj_ec_stripe_rec_nr(), thanks so much!